internal/handlers/category.html

```diff
@@ -0,0 +1,7 @@
+<h1>Category Page</h1>
+{{range .Categories}}
```

Comment:
try {{ range . }} here

Reply:
It worked.... Thanks

Reply:
What next?

Reply:
The data passed is .

Reply:
Do you understand why it worked? Next, let's re-write the HttpHandler for go fiber (make sure you are using /v2). Here is the documentation:

https://github.com/gofiber/fiber

Reply:
What I understand is that every data passed to a template can be accessed using {{ . }}